src/core/models/answersconfig.js

```diff
@@ -0,0 +1,216 @@
+/** @module AnswersConfig */
+
+import { LOCALE, PRODUCTION, QUERY_SOURCE, SANDBOX } from '../constants';
+import SearchConfig from '../models/searchconfig';
+import VerticalPagesConfig from '../models/verticalpagesconfig';
+
+const sandboxPrefix = `${SANDBOX}-`;
+const DEFAULTS = {
+  environment: PRODUCTION,
+  onReady: function () {},
+  useTemplates: true,
+  locale: LOCALE,
+  experienceVersion: 'PRODUCTION',
+  debug: false,
+  sessionTrackingEnabled: true,
+  onStateChange: function () {},
+  onVerticalSearch: function () {},
+  onUniversalSearch: function () {},
+  disableCssVariablesPonyfill: false,
+  querySource: QUERY_SOURCE
+};
+
+export default class AnswersConfig {
+  constructor (config) {
```

Comment:
Small nit: could we call this `rawConfig` or something like that? It just feels a bit weird for a class called `AnswersConfig` to be initialized with a `config`.